backend/src/govsg/middlewares/govsg-verification.middleware.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Response } from 'express'
+import { GovsgMessage } from '@govsg/models'
+import { GovsgVerification } from '@govsg/models/govsg-verification'
+
+export const listMessages = async (
+  req: Request,
+  res: Response
+): Promise<Response | void> => {
+  const { campaignId } = req.params
+  const { offset, limit } = req.query
+  const { rows, count } = await GovsgMessage.findAndCountAll({
+    where: {
+      campaignId: +campaignId,
+    },
+    offset: +(offset as string),
+    limit: +(limit as string),
+    include: [
+      {
+        model: GovsgVerification,
+        attributes: ['passcode'],
+      },
+    ],
+  })
+  const messages = rows.map((row) => {
+    const {
+      agency,
+      recipient,
+      officer_name,
+      recipient_name,
+      officer_designation,
+      ...remainingParams
+    } = row.params as any
+    console.debug(
```

Comment:
Hmm..., I meant it to avoid having to declare:

```
// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

But the intent isn't obvious at all, so let me remove it and add the necessary `eslint-disable`.

Reply:
Sorry I don't really understand. Why does `console.debug` avoid having to disable eslint?

Reply:
The `console.debug` "uses" the variable and would prevent the need to disable eslint on the `no-unused-vars` rule.